samples/cli-translator/src/main/resources/application.properties

```diff
@@ -1,5 +1,9 @@
+quarkus.langchain4j.chat-memory.memory-window.max-messages=2
```

Comment:
This is very awkward, isn't there a configuration to simply turn off memory altogether? I don't think there is. Perhaps this is something we should implement, @geoand?

Reply:
I didn't find one to turn it off, so I've restricted it to 2, to let system prompt and user prompt go through.

Reply:
There is: you can do it on `@RegisterAiService` with the `NoMemory...`  in `chatMemory...`

Reply:
Ah yeah. I mean this should be achievable by configuration properties.

Reply:
Yeah, I guess, but I'm generally in favor of having one way of doing things if we can

Reply:
Ok, so for now I'd suggest changing the AI service annotation to `@RegisterAiService(chatMemoryProviderSupplier = RegisterAiService.NoChatMemoryProviderSupplier.class)`, is that ok @dandreadis ? Feels cleaner than the config property.